payments-core/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -186,7 +186,7 @@ internal abstract class BaseAddPaymentMethodFragment(
                         selectedPaymentMethodResources.displayNameResource,
                         selectedPaymentMethodResources.iconResource,
                         this,
-                        false
+                        formFieldValues.saveForFutureUse
```

Comment:
Can you add a test verifying that the `PaymentSelection` is set on `sheetViewModel` with the correct value?

Reply:
done